Paymetheus.Rpc/Paymetheus.Rpc.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this entire file should remain untouched.  we absolutely do not want to depend on winforms from the rpc assembly